packages/server-wallet/src/engine/__test__/fixtures/test-channel.ts

```diff
@@ -313,10 +323,15 @@ export class TestChannel {
       await Channel.setLedger(this.channelId, this.startOutcome.assetHolderAddress, store.knex);
     }
 
-    const {fundingStrategy, fundingLedgerChannelId} = objective.data;
-    await Channel.query(store.knex)
-      .where({channelId: this.channelId})
-      .patch({fundingStrategy, fundingLedgerChannelId});
+    if (objective.data.fundingStrategy === 'Ledger') {
+      const {fundingStrategy, fundingLedgerChannelId} = objective.data;
+      await Channel.query(store.knex)
+        .where({channelId: this.channelId})
+        .patch({fundingStrategy, fundingLedgerChannelId});
+    } else {
+      const {fundingStrategy} = objective.data;
+      await Channel.query(store.knex).where({channelId: this.channelId}).patch({fundingStrategy});
+    }
```

Comment:
Maybe something like this is cleaner?
```suggestion
    const {fundingStrategy} = objective.data;
    let query = Channel.query(store.knex).where({channelId: this.channelId}).patch({fundingStrategy});
    if (objective.data.fundingStrategy === 'Ledger') {
        const {fundingLedgerChannelId} = objective.data;
        query.patch({fundingLedgerChannelId});
    }
    await query;
```